Dockerfile

```diff
@@ -21,11 +21,15 @@ RUN rm -rf postgres_install/build
 # net time waste in a lot of cases. Copying Cargo.lock with empty lib.rs should do the work.
 #
 FROM zenithdb/build:buster AS build
+
+ARG GIT_VERSION
+RUN if [ -z "$GIT_VERSION" ]; then echo "GIT_VERSION is reqired, use build_arg to pass it"; exit 1; fi
```

Comment:
I was thinking about checking it in `zenith_utils/src/lib.rs` so it would catch lack of `GIT_VERSION` (or empty one) in all cases, not just in this specific Dockerfile.

Reply:
This can quickly become quite tricky, and may require some sort of procmacro around it. Because GIT_VERSION won't be empty, it will contain prefix. So to check it we have to parse it back in compile time. It is simpler to check that after the build is complete (which is also not ideal), but again that goes inside multiple CI pipelines. I think current option is good enough. We can iterate on it in the future if we need to.

Reply:
Ah, I see. Do not have much experience with compile-time Rust, I thought it was easier.

Good enough, agree.